UefiTestingPkg/FunctionalSystemTests/MpManagement/Driver/MpManagement.inf

```diff
@@ -75,8 +76,9 @@
   gEfiLoadedImageProtocolGuid   ## CONSUMES
   gMpManagementProtocolGuid     ## PRODUCES
 
-[Protocols]
-  gEfiTimerArchProtocolGuid     ## CONSUMES
+[Protocols.AARCH64]
+  gEfiTimerArchProtocolGuid           ## CONSUMES
+  gEfiWatchdogTimerArchProtocolGuid   ## CONSUMES

```

Comment:
Does only the AARCH64 build require these?

Reply:
For now, yes. The requirement on x86 is unknown for now. We can try to coalesce them when we get there.